src/components/cards/EvaluationInfo.vue

```diff
@@ -139,39 +142,68 @@ export default class EvaluationInfo extends EvaluationInfoProps {
   }
 
   watch() {
-    console.log("Prop changed: " + this.filter);
+    console.log("Prop changed: " + this.file_filter);
```

Comment:
The devtools does this for you instead of requiring use of console.log: https://vuejs.org/v2/cookbook/debugging-in-vscode.html#Vue-Devtools